run_test.sh

```diff
@@ -10,7 +10,7 @@ if [[ ! -f $VENV ]]; then
         pip install --pre dbt-spark[ODBC] --upgrade
     elif [ $1 == 'synapse' ]
     then
-        pip install --pre dbt-synapse --upgrade
+        echo 'installing 0.19.0'; pip install --pre dbt-synapse>=0.19.0 --upgrade
```

Comment:
I've been having issues with using prerelease versions in CI, and it seems to be an issue with improperly reading `dbt_project.yml` config. In any case, I removed the `--pre` flag in #76 and CI ran with greater success as a result.